bin/buildBlueprints.js

```diff
@@ -130,40 +131,52 @@ console.log('...Reading Blueprints', argv.blueprintsPath);
 console.log('...cwd', process.cwd());
 
 var config = makeConfig(argv);
+var isTest = argv.target === TARGETS.TEST;
 
 build(config, function(stats) {
   if (stats.errors && stats.errors.length > 0 && !argv.watch) {
     console.log(colors.red('ERROR IN BUILD. Aborting.'));
     process.exit(1);
   }
 
-  if (argv.runTest) {
+  if (isTest) {
     console.log(colors.magenta(
       '\n   ******************************' +
       '\n   *       RUNNING TESTS        *' +
       '\n   ******************************'
     ));
 
     m = new Mocha({ reporter: mochaNotifier.decorate('spec') });
-    stats.assets.forEach(function(asset) {
-      m.addFile('./.test/' + asset.name);
+    glob(path.join(TEST_DIR, '/**/*.compiledtest'), function (err, files) {
```

Comment:
`stats.assets` only included the stats of the last build in the process since `build` chains a series of builds serially.  We want to add the files of all the test builds to mocha.